htdocs/core/modules/modStock.class.php

```diff
@@ -270,22 +271,22 @@ public function __construct($db)
 		$this->export_permission[$r]=array(array("stock","lire"));
 		$this->export_fields_array[$r]=array(
 			'e.rowid'=>'IdWarehouse','e.ref'=>'LocationSummary','e.description'=>'DescWareHouse','e.lieu'=>'LieuWareHouse','e.address'=>'Address','e.zip'=>'Zip',
-			'e.town'=>'Town','p.rowid'=>"ProductId",'p.ref'=>"Ref",'p.fk_product_type'=>"Type",'p.label'=>"Label",'p.description'=>"Description",'p.note'=>"Note",
-			'p.price'=>"Price",'p.tva_tx'=>'VAT','p.tosell'=>"OnSell",'p.tobuy'=>'OnBuy','p.duration'=>"Duration",'p.datec'=>'DateCreation',
-			'p.tms'=>'DateModification','sm.rowid'=>'MovementId','sm.value'=>'Qty','sm.datem'=>'DateMovement','sm.label'=>'MovementLabel',
+			'e.town'=>'Town','p.rowid'=>"ProductId",'p.ref'=>"Ref",'p.fk_product_type'=>"Type",'p.label'=>"Label",'p.description'=>"Description",'c.code'=>'Origin','p.note'=>"Note",
+			'p.price'=>"Price",'p.tva_tx'=>'VAT','p.tosell'=>"OnSell",'p.tobuy'=>'OnBuy','p.duration'=>"Duration",'p.net_measure'=>"NetMeasure",'p.customcode'=>"CustomCode",'p.datec'=>'DateCreation',
```

Comment:
Becasue the feature is conditionned by option PRODUCT_ADD_NET_MEASURE, having the field into array must also be aded only if PRODUCT_ADD_NET_MEASURE is on.